### ISSUE TITLE
Fetch AppImage from correct location during updates

### ISSUE THREAD

**Issue by adamkpickering** [rancher-sandbox/rancher-desktop#1624]
Currently, `LonghornProvider` only supports getting assets from github releases during an update. The AppImage package is currently hosted on OBS, which is an issue. There are two ways to go about this:
- Build the AppImage independently of OBS and upload it to github releases like the packages for Windows and macOS.
- Modify `LonghornProvider.checkForUpdates` to get the AppImage from OBS if running as an AppImage.

**Comment by jandubois** [comment]
> * Modify `LonghornProvider.checkForUpdates` to get the AppImage from OBS if running as an AppImage.

No, we should have all the assets in a single place, in the Github release.

**Comment by adamkpickering** [comment]
#1694 is related to this, and has been closed while I work on other things. It can be reopened once work resumes.
